src/main/resources/templates/db_scripts/schema.json

```diff
@@ -589,13 +589,27 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": false
```

Comment:
`"removeAccents": false` is correct because the accents are relevant for some libraries. Libraries that don't use accents in the call number are not affected by `"removeAccents": false`.
Can you change `"removeAccents"` from `true` to `false` for all call number fields?

Reply:
Sure, changed.